jqassistant.properties

```diff
@@ -1,21 +1,27 @@
 category=External Analyzers
 description=Analyze issues reported by jQAssistant.
 homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
-archivedVersions=1.7.2
-publicVersions=1.7.3
+archivedVersions=1.7.3
+publicVersions=1.8.1
  
 defaults.mavenGroupId=org.jqassistant.contrib.sonarqube
 defaults.mavenArtifactId=sonar-jqassistant-plugin
 
+1.8.1.description=Upgraded to jQAssistant 1.8.0 
```

Comment:
I suppose it's academic, but what happened to 1.8.0? As a user I find this gap a little odd. (Not a deal-breaker, just odd.)

Reply:
After releasing 1.8.0 to Maven Central I noticed that the quality gate on sonarcloud.io was red. The reason: the setup for test coverage was still based on the JaCoCo binary format. needed to upgrade it to the XML reports.